internal/pkg/otel/samples/linux/gateway.yml

```diff
@@ -0,0 +1,100 @@
+receivers:
+  # Receives data from other Collectors in Agent mode
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317 # Listen on all interfaces
+      http:
+        endpoint: 0.0.0.0:4318 # Listen on all interfaces
+
+connectors:
+  # Routing connector splits infra metrics from other metrics and routes them to ECS-based pipeline
+  # Other metrics are exported in OTel-native format to Elasticsearch
+  routing:
+    default_pipelines: [metrics/otel]
+    error_mode: ignore
+    table:
+      - context: metric
+        statement: IsMatch(instrumentation_scope.name, "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/*")
```

Comment:
Looks like the collector does not like this syntax:

```
Error: failed to build pipelines: editor names must start with a lowercase letter but got 'IsMatch'
2025/05/27 10:13:29 collector server run finished with error: failed to build pipelines: editor names must start with a lowercase letter but got 'IsMatch'
```

Checking how can we fix it